02_process/src/prepData_fun.R

```diff
@@ -0,0 +1,28 @@
+# Prepare the data for plotting
+
+
+prepData <- function(data_inPath, savePath) {
+  
+  require(dplyr)
+  require(readr)
+  require(stringr)
```

Comment:
Same idea here, I would change these to `library()` calls instead and then move right above `prepData()` in this script.